app/src/main/java/com/ahgitdevelopment/course/customexamples/navigation/AppNavigation.kt

```diff
@@ -32,5 +33,8 @@ fun AppNavigation() {
         ) {
             ScreenB(navController, it.arguments?.getString("text"))
         }
+        composable(route = AppScreens.ExampleListScreen.route) {
```

Comment:
You've kept `ExampleListScreen.route` instead of something like: 'LazyListScreen.route`